docs/examples/bitops-deploy-heyemoji/README.md

```diff
@@ -0,0 +1,16 @@
+# Bitops Operations Repo
```

Comment:
@PhillypHenning any idea why there are a bunch of `bitops-deploy-heyemoji` files in here?